python/ray/tests/test_placement_group.py

```diff
@@ -791,5 +792,60 @@ def ping(self):
     assert all(ray.get([a.ping.remote() for a in actors]))
 
 
+def test_capture_child_tasks(ray_start_cluster):
+    cluster = ray_start_cluster
+    total_num_actors = 4
+    for _ in range(2):
+        cluster.add_node(num_cpus=total_num_actors)
+    ray.init(address=cluster.address)
+
+    pg = ray.util.placement_group(
+        [{
+            "CPU": 2
+        }, {
+            "CPU": 2
+        }], strategy="STRICT_PACK")
+    ray.get(pg.ready(), timeout=1)
+
+    # If get_current_placement_group is used when the current worker/driver
+    # doesn't belong to any of placement group, it should raise an exception.
+    with pytest.raises(RuntimeError):
+        get_current_placement_group()
```

Comment:
Why not return None instead?

Reply:
The initial reason was I thought when users do `pg.bundle_spec` or `pg.ready` after this API, it will raise a weird exception. None makes sense given use case of this API will be mostly inside `options` method.

Reply:
I will return None and make it clear in the documentation.